### DISCUSSION TITLE
InvalidInputException on estimate_u_using_random_sampling()

### DISCUSSION THREAD

**Discussion by mmagoffin-sd** [moj-analytical-services/splink#842] [Q&A] [answered]
Hello, 

When trying to run linker.estimate_u_using_random_sampling(target_rows=8000000), I get the following error:

![Screen Shot 2022-10-11 at 3 00 06 PM](https://user-images.githubusercontent.com/85239124/195176680-f8fcd97c-2c96-468a-96d7-60c296ba85d1.png)
![Screen Shot 2022-10-11 at 3 00 13 PM](https://user-images.githubusercontent.com/85239124/195176688-0e886299-1364-4949-bcbe-ef88761a7ab0.png)

Any ideas to where the root of this error could be?

**Comment by RobinL** [comment, marked as answer]
This is likely to be something to do with your settings dictionary - specifically the comparisons your setting.  If you post your full code, we might be able to help further
